programs/programs/il/example_program/calculator.py

```diff
@@ -0,0 +1,14 @@
+from programs.programs.calc import MemberEligibility, ProgramCalculator
+
+
+# NOTE: This an example program where each child in the household would get $750
+class ExampleCalculator(ProgramCalculator):
```

Comment:
trying to get things to show up on the results page. ignore this file.

TODO: delete this